code/game/objects/structures/vehicles/wheelchair.dm

```diff
@@ -220,13 +220,13 @@
 		internal_battery.use(2) //Example use: 100 charge to get from the cargo desk to medbay side entrance
```

Comment:
That's not quite right. The wheelchair doesn't lock up entirely if out of batteries, you can still move it by hand (but it's even slower than a normal wheelchair).